code/ship/shipfx.cpp

```diff
@@ -707,6 +708,7 @@ void shipfx_warpout_start( object *objp )
 		return;
 	}
 
+	Assertion(shipp->warpin_effect != nullptr,"shipfx_warpout_start() was fed a ship with an uninitialized warpin_effect.");
```

Comment:
You need to check for `warpout_effect` here, not `warpin_effect`.  Also, as an extremely minor nitpick, put a space between the comma and the quote.

Reply:
Thanks!  Should be good now.